app/assets/stylesheets/show_page.scss

```diff
@@ -1,4 +1,55 @@
+$celtics: #195a07;
+
 .meetup-avatar {
-  height: 50px;
-  width: 50px;
+  height: 100%;
+  width: 100%;
```

Comment:
Properties should be ordered height, max-height, max-width, min-height, min-width, transform, width